vendor/bat-native-ledger/src/bat/ledger/internal/bat_helper.h

```diff
@@ -417,6 +416,9 @@ bool getJSONResponse(const std::string& json,
 bool getJSONAddresses(const std::string& json,
                       std::map<std::string, std::string>* addresses);
 
+bool getJSONMessage(const std::string& json,
```

Comment:
Please use Pascal Case, i.e. `GetJSONMessage` for function names. Non pascal case function names are legacy from the beginning of ledger

Reply:
This requires whole class refactor and not related to this PR

Reply:
@samartnik For new code we follow coding style from Chromium/Google. Legacy code is refactored as/when we work in those areas

Reply:
NONCRITICAL:
covered in :  https://github.com/brave/brave-browser/issues/6235 (22)